helm/harbor-scanner-trivy/values.yaml

```diff
@@ -4,7 +4,7 @@ fullnameOverride: ""
 image:
   registry: docker.io
   repository: aquasec/harbor-scanner-trivy
-  tag: 0.24.0
+  tag: 0.25.0
```

Comment:
This is out of scope. We do release prepare patches to bump up Helm chart and image tags